pytest_reana/version.py

```diff
@@ -14,4 +14,4 @@
 
 from __future__ import absolute_import, print_function
 
-__version__ = "0.4.0"
+__version__ = "0.4.1.dev20181106"
```

Comment:
Let's use 0.4.1 directly...

Reply:
Ok, amended `README.rst` and `CHANGES.rst`